dev-resources/com/walmartlabs/lacinia/test_utils.clj

```diff
@@ -196,8 +197,8 @@
 
 (defmacro expect-message
   [expected]
-  `(is (= ~expected
-          (<message!!))))
+  `(is (= (<message!!)
```

Comment:
Why this change?  Convention and tooling dictates that the expected value goes first, actual value second.

Reply:
I noticed it was printing inverted.  Right or wrong, it probably warrants a separate commit so I'll drop this.

Reply:
@hlship i pushed an update that drops the hunk discussed above.